src/components/Modal/Example.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <div>
+    <button @click="openModal()">coucou</button>
```

Comment:
```suggestion
    <button @click="openModal()">X</button>
```

Le must serai une croix avec 2 span bien propre et animés